src/main/java/bio/terra/service/dataset/DatasetRequestValidator.java

```diff
@@ -132,7 +137,7 @@ private void validateIntPartitionOptions(IntPartitionOptionsModel options,
                 columns.stream().filter(c -> targetColumn.equals(c.getName())).findFirst();
 
             if (matchingColumn.isPresent()) {
-                String colType = matchingColumn.get().getDatatype();
+                String colType = matchingColumn.get().getDatatype().toString();
 
                 if (!"INTEGER".equalsIgnoreCase(colType) && !"INT64".equalsIgnoreCase(colType)) {
```

Comment:
I believe this can also be changed to use the enum types instead of strings.

Reply:
oops missed that one, thanks!